lib/player.js

```diff
@@ -266,6 +266,17 @@ goog.require('shaka.util.StreamUtils');
  * @exportDoc
  */
 
+/**
+ * @event shaka.Player.SegmentParsedEvent
+ * @description Optionally fired from inside a manifest parser plugin if a newly
+ *   parsed segment has some extra metadata. It emits mime type of the parser,
+ *   id of the stream that segment belonged to, and segment data (format may
+ *   differ according to manifest parser).
+ * @property {string} type
+ *   'segmentparsed'
+ * @exportDoc
```

Comment:
The data values to the event are given @property tags, rather than just mentioned in text in the @description. See, for instance, the documentation for `shaka.Player.LargeGapEvent`.

Reply:
Followed the format and jsdoc for LargeGapEvent.